portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,45 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+
```

Comment:
This new line can be removed since it's not really adding to the readability of the file (same with other extra lines in this file). Generally, new lines between elements are used to improve the readability of the file by grouping larger, logical chunks of the template together (like if you had a large section sharing about your cooking experiences).

You might need to modify some settings in your IDE that could be adding these lines in automatically!

Reply:
Got it. I've removed the unnecessary newlines